### PR TITLE
✨ Add more levels

### PR DESCRIPTION
**Pull request by maelehrhard**
✨ Add up to level 5
✨ Change level when completed (up to level 5)

### CODE REVIEW

File: arkanoid.c [hunk 1]
```diff
@@ -243,6 +245,14 @@ void update_level() {
             }
         }
     }
+    if (is_level_completed()) {
+        level_number++;
+        if (level_number <= MAX_LEVELS) {
+            char filename[20];
+            sprintf(filename, "levels/%d.level", level_number);
+            load_level(filename);
+        }
```
Comment (axelmunch) [inline arkanoid.c]: A function such as `load_next_level()` in levels.c would add abstraction

File: arkanoid.c [hunk 2]
```diff
@@ -12,6 +12,7 @@
 
 Ball ball;
 VAUS vaus;
+uint8_t level_number;
```
Comment (axelmunch) [inline arkanoid.c]: This would be better in levels.c script
levels.c should manage everything revolving around levels

File: arkanoid.c (hunk 1 above)
Comment (maelehrhard) [reply arkanoid.c]: ok done 👍

File: arkanoid.c (hunk 2 above)
Comment (maelehrhard) [reply arkanoid.c]: RIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIGHT 😎

**Review by axelmunch — APPROVED** [review]

**Pull request merged**